src/tracker/amigaversion.h.in

```diff
@@ -1,3 +1,4 @@
 static const char __attribute((used)) *amiga_ver = "$VER: MilkyTracker ${VER_X}.${VER_YY} (${VER_DAY}.${VER_MONTH}.${VER_YEAR}) MilkyTracker v${VER_X}.${VER_YY}.${VER_ZZ} ported by AmigaDev.com Team\0";
```

Comment:
this is VER_YEAR etc is used from main CMakeLists.txt

Reply:
OK